xml/System.Runtime.InteropServices/StructLayoutAttribute.xml

```diff
@@ -215,7 +215,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- If the `CharSet` field is set to <xref:System.Runtime.InteropServices.CharSet.Unicode?displayProperty=nameWithType>, all string arguments are converted to Unicode characters (`LPWSTR`) before they are passed to the unmanaged implementation. If the field is set to <xref:System.Runtime.InteropServices.CharSet.Ansi?displayProperty=nameWithType>, the strings are converted to ANSI strings (`LPSTR`). If the `CharSet` field is set to <xref:System.Runtime.InteropServices.CharSet.Auto?displayProperty=nameWithType>, the conversion is platform-dependent (ANSI on Windows 98 and Windows Me, and Unicode on later versions).  
+ If the `CharSet` field is set to <xref:System.Runtime.InteropServices.CharSet.Unicode?displayProperty=nameWithType>, all string arguments are converted to Unicode characters (`LPWSTR`) before they are passed to the unmanaged implementation. If the field is set to <xref:System.Runtime.InteropServices.CharSet.Ansi?displayProperty=nameWithType>, the strings are converted to ANSI strings (`LPSTR`). If the `CharSet` field is set to <xref:System.Runtime.InteropServices.CharSet.Auto?displayProperty=nameWithType>, the conversion is platform-dependent (ANSI on Windows 98 and Windows Me, and Unicode on later OS versions).  
```

Comment:
>ANSI on Windows 98 and Windows Me [](start = 551, length = 33)

Remove?

Reply:
Perhaps, but then the comment for `CharSet.Auto` would say that the conversion is always Unicode, which is no different to specifying `CharSet.Unicode`. Perhaps I should just leave `Auto` out of the remark?

Reply:
> Perhaps I should just leave Auto out of the remark?

Perhaps include Auto with Unicode: "If the CharSet field is set to System.Runtime.InteropServices.CharSet.Unicode or ...CharSet.Auto, all string arguments are converted to Unicode ..."

---
In reply to: [599135072](https://github.com/dotnet/dotnet-api-docs/pull/5453#discussion_r599135072) [](ancestors = 599135072)